README.md

```diff
@@ -133,6 +134,31 @@ python analyze.py --config_file config.ini --log_level DEBUG
 
 See [Generic Command Line Parameters](#generic-command-line-parameters) for more details.
 
+# Analysis with sclite
+`sclite` is an open source tool designed to evaluate STT transcription results. More information about `sclite` can be found here -- https://people.csail.mit.edu/joe/sctk-1.2/doc/sclite.htm#sclite_name_0.
```

Comment:
Can you add one sentence why someone would run this extra analysis (presumably new&better reports)?